test/errors/test_FPDF_errors.py

```diff
@@ -107,6 +109,14 @@ def test_repeated_calls_to_output(tmp_path):
     assert_pdf_equal(pdf, HERE / "repeated_calls_to_output.pdf", tmp_path)
 
 
+def test_unsupported_image_filter_error():
+    img = Image.open(HERE / "flowers.png")
+    image_filter = "N/A"
+    with pytest.raises(FPDFException) as error:
+        _to_data(img, image_filter)
```

Comment:
`_to_data` is currently a module-private function: could we instead test the parent `get_img_info()` public function?

Reply:
Yes right! I will keep in mind next times: better to test public functions instead of  module-private functions